css/Z_INDEX.md

```diff
@@ -121,9 +121,6 @@
 `.i-amphtml-image-slider-right-mask` | 1 | [extensions/amp-image-slider/0.1/amp-image-slider.css](/extensions/amp-image-slider/0.1/amp-image-slider.css)
 `.i-amphtml-inline-gallery-pagination-count` | 1 | [extensions/amp-inline-gallery/0.1/amp-inline-gallery-pagination.css](/extensions/amp-inline-gallery/0.1/amp-inline-gallery-pagination.css)
 `.i-amphtml-inline-gallery-pagination-dot-container` | 1 | [extensions/amp-inline-gallery/0.1/amp-inline-gallery-pagination.css](/extensions/amp-inline-gallery/0.1/amp-inline-gallery-pagination.css)
-`container['&$inset']` | 1 | [extensions/amp-inline-gallery/1.0/pagination.jss.js](/extensions/amp-inline-gallery/1.0/pagination.jss.js)
```

Comment:
Hmm.. Yes, they should be updated.  Thanks for calling it out.  Since this file is auto-generated, I wonder if there's something else missing somewhere with regard to `pagination.jss`, investigating now.

Reply:
Hmm, re-generating the file, removed all the deletions of `pagination`, odd.  Not sure what happened the first time.  This version looks correct!